### ISSUE TITLE
Login/out messages for clan discord 

### ISSUE THREAD

**Issue by TomLewis** [RoinujNosde/SimpleClans#415]
**Is your feature request related to a problem? Please describe.**
Nope, players just want to see when people from their crew login, I'm their clan discord 

**Describe the solution you'd like**
Listen to the login and out event and record clan members activity into their clan discords 

**Describe alternatives you've considered**
Figure out if there's a command in simple clans to send messages to a clan discord and somehow listen to the event

**Comment by Nishikiyama** [comment]
Are they discord channels or servers? 
In the case of channels, just use DiscordSRV, guide to that [here](https://wiki.roinujnosde.me/simpleclans/integration/discordsrv-integration), and the download to DiscordSRV [here](https://www.spigotmc.org/resources/discordsrv.18494/).

You could just have them send a message in clan chat when they login. You can read up more on how SRV works [here](https://docs.discordsrv.com/master/) and for the simpleclans part [here](https://wiki.roinujnosde.me/simpleclans/integration/discordsrv-integration)

**Comment by TomLewis** [comment]
I use DiscordSRV & already have a working SimpleClans Discord setup, my request is to expand on the SimpleClans setup to mearly show sessions into the respected clan channels.

Your comment is just fluff.

**Comment by Nishikiyama** [comment]
I mean the feature imo is kinda useless, if you already have a discord with your clan members, you know how to communicate, no? Adding a feature that is technically just trashcode makes kind of no sense, learn to communicate 😉

**Comment by TomLewis** [comment]
Obviously, being able to look at a log of when players log on for your crew via mobile is extreamly helpful when you have a crew of say 100 people. 

Just because you do not have a use case, does not mean others woudlnt find this extreamly helpful. 

Its litearlly a line of code "trashcode".

**Comment by Nishikiyama** [comment]
You have Java knowledge, don't you? Create it yourself; I consider it trash code because not a single other person has asked for such a feature -> it is practically unneeded.

**Comment by TomLewis** [comment]
Why are you replying in this repository? You are nothing to do with this ticket or project.

You don't even have a server

**Comment by Nishikiyama** [comment]
Why are you so aggressive? You clearly have no idea of my presence or any kind of knowledge about me. You’re just guessing. Only thing I told you was that it’s not worth coding for one person to use, if every request was coded in, we’d be working 48 hours a day on requests that wouldn’t be useful. I’m simply giving you a reality check, yet you’re being hostile.

**Comment by Tomut0** [comment]
Please be more respectful to each other. It's not the place to argue. 

Personally me and @RoinujNosde find this suggestion lovely. I also think the notification can be send when a player joins or leaves a clan or even in other events. 

As RoinujNosde noticed, players logins/logouts would cause to massive spam of messages. He suggested to pin the notification. 

@TomLewis your suggestion isn't bad. However, we can't be sure **when** it will (and will it be at all) be implemented. Since @Nishikiyama is a contributer, the contributer **can** write your idea. But because of the argue I don't think it's possible.

@Nishikiyama Linus Torvalds once said that C++ is "bullshit". He didn't see the perspective of the language. That's there your go. SimpleClans wasn't updated with fresh ideas for a long time, but instead of do something, we are sitting here and arguing. 

If I'll see one more provocative reply, I'll instantly close the issue and we will **never** come back to it. Thanks.

**Comment by TomLewis** [comment]
@Tomut0 thanks! 
Players like being able to see when events happen outside of server playing times, so more events would be helpful, act as a form of log for them to catch up on.

At present it's just chat, but if it's treated more like a clan events log, including logins this would be fantastic, the crews could toggle their events If they think it's too busy.
Maybe best as "opt in".

Personally on my server, we have crews that only talk in crew chat, rarely use global, so for those crews it wouldn't be spam, but definitely in small 5 person crews it may be. But, if it's treated like a log, that's the point, information!

I did previously request a change to the BB system to move it into its own table outside of the clan row in mysql, if that ever happens then this could be in the same ballpark area of togetherness.
